src/shared/constants/DeveloperUtils.js

```diff
@@ -0,0 +1,81 @@
+import { compareSync } from 'bcrypt';
```

Comment:
this file isn't a constants file - rather this file should live closer to the developers controller file. Yous should create a new `developers` directory that includes `developers.js` and `utils.js` where the utils file includes all these functions

Reply:
Okay